src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -0,0 +1,143 @@
+#include "benchmark_table_encoder.hpp"
+
+#include "constant_mappings.hpp"
+#include "encoding_config.hpp"
+#include "resolve_type.hpp"
+#include "storage/base_encoded_segment.hpp"
+#include "storage/base_value_segment.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "storage/table.hpp"
+#include "types.hpp"
+
+namespace {
+
+using namespace opossum;  // NOLINT
+
+SegmentEncodingSpec generate_segment_encoding_spec(const BaseValueSegment&) { return {EncodingType::Unencoded}; }
```

Comment:
I don't like the term "generate" here. It sounds too much like performing actual work. Why not simply `get_segment_encoding_spec`?